server/protocols/jmap/src/test/java/org/apache/james/jmap/AccessTokenAuthenticationStrategyTest.java

```diff
@@ -40,29 +42,39 @@
 import org.junit.Test;
 import org.slf4j.Logger;
 
+import com.google.common.collect.ImmutableList;
+
 public class AccessTokenAuthenticationStrategyTest {
 
     private AccessTokenManagerImpl mockedAccessTokenManager;
     private MailboxManager mockedMailboxManager;
     private AccessTokenAuthenticationStrategy testee;
+    private HttpServletRequest request;
```

Comment:
you would not have to mess with HttpServletRequest if you don't use HeadersAuthenticationExtractor as a static reference